src/dags/sport.py

```diff
@@ -14,13 +14,15 @@
 from typeahead_location_operator import TypeAHeadLocationOperator
```

Comment:
This file does not really belong to `http_fetch_operator` ;)

Reply:
That's true. Because the sport dataset benefits from the http_fetch_operator (one source of the sports dataset is not loaded without the workarround) and there is also a request for some minor adjustements, I combined the two in one PR. Maybe I should have split them up for a cleaner track